views/js/controller/Delivery/monitoring.js

```diff
@@ -144,29 +144,29 @@ define([
 
             // request the server to authorise the selected delivery executions
             function authorise(selection) {
-                execBulkAction('authorize', __('Authorize Delivery Session'), selection, function(selection, reason){
-                    request(authoriseUrl, selection, reason, __('Delivery executions have been authorised'));
+                execBulkAction('authorize', __('Authorize Session'), selection, function(selection, reason){
+                    request(authoriseUrl, selection, reason, __('Session authorized'));
```

Comment:
Maybe not a single session has been authorized? All other sentences use plural, why not this one?